keps/sig-node/3317-user-namespaces/README.md

```diff
@@ -0,0 +1,1015 @@
+  more details to the KEP (like CRI changes, changes to container runtimes, how to
+  configure kubelet ranges, etc.), and then the Windows folks can review and we
+  adjust as needed (I doubt it will be a big change, if any). After that we switch
+  the KEP to implementable (or if there are long delays to get a review, we might
+  decide to do it after the first alpha, as the community prefers and time
+  allows). Same applies for VM runtimes.
+  UPDATE: Windows maintainers reviewed and [this change looks good to them][windows-review].
+
+[windows-review]: https://github.com/kubernetes/enhancements/pull/3275#issuecomment-1121603308
+
+### Summary of the Proposed Changes
+
+[This table](https://docs.google.com/presentation/d/1z4oiZ7v4DjWpZQI2kbFbI8Q6botFaA07KJYaKA-vZpg/edit#slide=id.gfd10976c8b_1_41) gives you a quick overview of each phase (note it is outdated, but still useful for a general overview).
+
+### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
+
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Additionally, for Alpha try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+-->
+
+Most of the changes will be in a new file we will create, the userns manager.
+Tests are already written for that file.
+
+The rest of the changes are more about hooking it up so it is called.
+
+- `pkg/kubelet/kuberuntime/kuberuntime_container_linux.go`: 24.05.2022 - 90.8%
+- `pkg/volume/util/operationexecutor/operation_generator.go`: 24.05.2022 - 8.6%
```

Comment:
This seems super low.

Can we first increase the coverage of this package before we start changing this?

Reply:
that seems out of scope to me.  We are just plumbing some values when we call `volumeMounter.SetUp()` without adding any additional feature to the file